src/tests/liking.accommodation.spec.js

```diff
@@ -67,4 +67,22 @@ describe('Like/Unlike an accomodation facility', () => {
         done();
       });
   });
+  it('it should return 200  and count the number of likes of the accommodation', (done) => {
+    chai.request(app)
+      .get('/api/v1/accommodations/2/like')
+      .set('Authorization', helper.createToken(2, 'requester@gmail.com', true, 'requester'))
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
```

Comment:
You need to also expect that the number of likes is correct

Reply:
great